### ISSUE TITLE
Get distinct-attribute returns an int instead of the attribute name

### ISSUE THREAD

**Issue by eskombro** [meilisearch/MeiliSearch#804]
**Describe the bug**
When you set a `distinct attribute` on MeiliSearch, and you do a GET request on `/indexes/{{indexUID}}/settings/distinct-attribute`, the server responds with an integer instead of `null` or the attribute name

**To Reproduce**
1. Create an index
2. Update distinct-attribute with an attribute name
3. Get distinct-attribute
4. Response displays an int, not the attribute name.

**Expected behavior**
I am supposed to receive the attribute name as described [in the docs](https://docs.meilisearch.com/references/distinct_attribute.html#get-distinct-attribute)

**Screenshots**
![Screenshot 2020-06-29 at 04 35 01](https://user-images.githubusercontent.com/10339833/85967099-ecbf2d00-b9c1-11ea-8d33-88b9dc013861.png)

Tested on docker latest

**Comment by MarinPostma** [comment]
hey, that's a duplicate of #798 that should be fixed once #800 is merged :)
